### PR TITLE
Rloutlaw/java jdk official guidance

### PR DESCRIPTION
**Pull request by rloutlaw**
Don't merge-still working through this.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/java-supported-jdk-runtime.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-supported-jdk-runtime.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) | [Details](#user-content-docs-ref-conceptual%2Fjava-supported-jdk-runtime.md)
[docs-ref-conceptual/index.yml](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/index?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/index?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/java-azure-tools.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-azure-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-azure-tools?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-azure-tools?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/media/azure-jdk-support.png](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/media/azure-jdk-support.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable) |
[docs-ref-conceptual/toc.yml](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable) |

<a name="docs-ref-conceptual%2Fjava-supported-jdk-runtime.md"></a>
### [docs-ref-conceptual/java-supported-jdk-runtime.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-supported-jdk-runtime.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/docs-ref-conceptual/azure/azure-functions/functions-create-first-java-maven).
 - **Line 41**: **[Warning]** Invalid file link:(~/docs-ref-conceptual/azure/azure-functions/functions-create-first-java-maven).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C16%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201810160005473433-666%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=IMQVPUl6L0cRYPdgt4E5YBXvVVE0096cYVdsSdUZ0jI%3D&st=2018-10-16T00%3A58%3A32Z&se=2018-11-16T01%3A03%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/java-supported-jdk-runtime.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-supported-jdk-runtime.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) | [Details](#user-content-docs-ref-conceptual%2Fjava-supported-jdk-runtime.md)
[docs-ref-conceptual/index.yml](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/index?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/index?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/java-azure-tools.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-azure-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-azure-tools?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-azure-tools?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/media/azure-jdk-support.png](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/media/azure-jdk-support.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable) |
[docs-ref-conceptual/toc.yml](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable) |

<a name="docs-ref-conceptual%2Fjava-supported-jdk-runtime.md"></a>
### [docs-ref-conceptual/java-supported-jdk-runtime.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-supported-jdk-runtime.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/docs-ref-conceptual/azure/azure-functions/functions-create-first-java-maven).
 - **Line 41**: **[Warning]** Invalid file link:(~/docs-ref-conceptual/azure/azure-functions/functions-create-first-java-maven).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201810162336499886-666%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wlFrasOkz5zUU6VnsNFX9Twf7Iwcb2uerS7axwc9fSA%3D&st=2018-10-17T00%3A30%3A53Z&se=2018-11-17T00%3A35%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/index.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ sections:
         src: https://docs.microsoft.com/java/docs-ref-conceptual/media/spring.svg
       title: Spring on Azure
     - href: java-azure-tools.md
-      html: <p>IDE plugins &amp; Tools</p>
+      html: <p>Supported JDKs &amp; IDE plugins</p>
```
Comment (brunoborges) [inline docs-ref-conceptual/index.yml]: Wdyt of "Tools, IDEs, and JDKs"

File: docs-ref-conceptual/toc.yml [hunk 2]
```diff
@@ -122,6 +122,8 @@
       items:
         - name: Overview
           href: java-azure-tools.md
+        - name: Supported Azure JDKs
```
Comment (brunoborges) [inline docs-ref-conceptual/toc.yml]: I prefer to call this "Supported Java JDKs". We do not have an "Azure JDK".

Comment (brunoborges) [reply docs-ref-conceptual/toc.yml]: The document should also say that customers can run other JDKs, but commercial support is provided automatically if they use Azul's

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/index.yml](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/index?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/index?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/java-azure-tools.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-azure-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-azure-tools?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-azure-tools?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/java-supported-jdk-runtime.md](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/java-supported-jdk-runtime.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview) |
[docs-ref-conceptual/media/azure-jdk-support.png](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/media/azure-jdk-support.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable) |
[docs-ref-conceptual/toc.yml](https://github.com/rloutlaw/azure-docs-sdk-java-1/blob/rloutlaw/java-jdk-official-guidance/docs-ref-conceptual/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-preview&branch=pr-en-us-666) (azure-java-preview)<br/> [View](https://review.docs.microsoft.com/en-us/java/azure/java-supported-jdk-runtime?view=azure-java-stable&branch=pr-en-us-666) (azure-java-stable) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C17%5C5de83821-f44b-0e93-473e-c19393f102fe%5CPullRequest%5C201810171717301463-666%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JePL1yfmtEnZWtXfBaNXMQuhFUKEJQn2ZEuVh7EE6bI%3D&st=2018-10-17T18%3A10%3A34Z&se=2018-11-17T18%3A15%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/index.yml (hunk 1 above)
Comment (rloutlaw) [reply docs-ref-conceptual/index.yml]: Great change. Made it into the current version

**Pull request merged**